random/module.py

```diff
@@ -0,0 +1,210 @@
+import aiohttp
+import random
+
+import discord
+from discord.ext import commands
+
+from core import utils, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Random(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def random(self, ctx, first: int, second: int = 0):
+        """Generate random number within the interval"""
+        if first > second:
+            first, second = second, first
+
+        await ctx.reply(random.randint(first, second))
+
+    @commands.cooldown(rate=3, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def pick(self, ctx, *args):
+        """Pick an option"""
+        for i, arg in enumerate(args):
+            if arg.endswith("?"):
+                args = args[i + 1:]
+                break
+
+        if not len(args):
+            return
+
+        option = utils.Text.sanitise(random.choice(args))
+        if option is not None:
+            await ctx.reply(option)
+
+    @commands.cooldown(rate=3, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def flip(self, ctx):
+        """Yes/No"""
+        choices = [_(ctx, "Yes"), _(ctx, "No")]
+        await ctx.reply(random.choice(choices))
+
+    @commands.cooldown(rate=5, per=20, type=commands.BucketType.channel)
+    @commands.command(aliases=["unsplash"])
+    async def picsum(self, ctx, *, seed: str = None):
```

Comment:
This function is longer than ~10 lines, can you add type hints to the variables? There's no need to do that to other shorter functions above, but I'm not against either.